clang/test/Parser/cxx0x-attributes.cpp

```diff
@@ -451,3 +451,5 @@ namespace P2361 {
                                  // expected-warning {{use of the 'deprecated' attribute is a C++14 extension}}
 [[nodiscard("\123")]] int b(); // expected-error{{invalid escape sequence '\123' in an unevaluated string literal}}
 }
+
+alignas(int) struct AlignAsAttribute {}; // expected-error {{misplaced attributes; expected attributes here}}
```

Comment:
Note to other reviewers, this C++ test exhibits the same behavior on trunk already: https://godbolt.org/z/s5T3hfxsr